ddtrace/contrib/aiohttp/patch.py

```diff
@@ -1,158 +1,4 @@
-            result = await self.__wrapped__._create_connection(req, *args, **kwargs)
-            return result
-
-
-@with_traced_module
-async def _traced_clientsession_request(aiohttp, pin, func, instance, args, kwargs):
-    method = get_argument_value(args, kwargs, 0, "method")  # type: str
-    url = URL(get_argument_value(args, kwargs, 1, "url"))  # type: URL
-    params = kwargs.get("params")
-    headers = kwargs.get("headers") or {}
-
-    with pin.tracer.trace(
-        schematize_url_operation("aiohttp.request", protocol="http", direction=SpanDirection.OUTBOUND),
-        span_type=SpanTypes.HTTP,
-        service=ext_service(pin, config.aiohttp_client),
-    ) as span:
-        if config.aiohttp_client.split_by_domain:
-            span.service = url.host
-
-        if pin._config["distributed_tracing"]:
-            HTTPPropagator.inject(span.context, headers)
-            kwargs["headers"] = headers
-
-        span.set_tag_str(COMPONENT, config.aiohttp_client.integration_name)
-
-        # set span.kind tag equal to type of request
-        span.set_tag_str(SPAN_KIND, SpanKind.CLIENT)
-
-        # Params can be included separate of the URL so the URL has to be constructed
-        # with the passed params.
-        url_str = str(url.update_query(params) if params else url)
-        host, query = extract_netloc_and_query_info_from_url(url_str)
-        set_http_meta(
-            span,
-            config.aiohttp_client,
-            method=method,
-            url=str(url),
-            target_host=host,
-            query=query,
-            request_headers=headers,
-        )
-        resp = await func(*args, **kwargs)  # type: aiohttp.ClientResponse
-        set_http_meta(
-            span, config.aiohttp_client, response_headers=resp.headers, status_code=resp.status, status_msg=resp.reason
-        )
-        return resp
-
-
-@with_traced_module_sync
-def _traced_clientsession_init(aiohttp, pin, func, instance, args, kwargs):
-    func(*args, **kwargs)
-    instance._connector = _WrappedConnectorClass(instance._connector, pin)
-
-
-def _patch_client(aiohttp):
-    Pin().onto(aiohttp)
-    pin = Pin(_config=config.aiohttp_client.copy())
-    pin.onto(aiohttp.ClientSession)
-
-    wrap("aiohttp", "ClientSession.__init__", _traced_clientsession_init(aiohttp))
-    wrap("aiohttp", "ClientSession._request", _traced_clientsession_request(aiohttp))
-
-
-def patch():
-    import aiohttp
-
-    if getattr(aiohttp, "_datadog_patch", False):
-        return
-
-    _patch_client(aiohttp)
-
-    aiohttp._datadog_patch = True
-
-
-def _unpatch_client(aiohttp):
-    unwrap(aiohttp.ClientSession, "__init__")
-    unwrap(aiohttp.ClientSession, "_request")
-
-
-def unpatch():
-    import aiohttp
-
-    if not getattr(aiohttp, "_datadog_patch", False):
-        return
-
-    _unpatch_client(aiohttp)
-
-    aiohttp._datadog_patch = False
+# TODO: deprecate and remove this module
```

Comment:
@rachelyangdog In this PR we do not deprecate the patch.py module.  This module is imported in `ddtrace.contrib.aiohttp.__init__.py`. `ddtrace.contrib.aiohttp.__init__.py` is imported when the aiohttp integration is enabled. We do not want to log a deprecation warning when our library is used as expected. 

We can deprecate this module in a future PR.